### ISSUE TITLE
BigQuerySampleApplicationIntegrationTests failing with NoSuchMethodError

### ISSUE THREAD

**Issue by ldetmer** [GoogleCloudPlatform/spring-cloud-gcp#3140]
[BigQuerySampleApplicationIntegrationTests are failing with error below](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/actions/runs/10449601755/job/28932251013#step:10:10242)

```
java.lang.NoSuchMethodError: 'io.opentelemetry.api.metrics.LongHistogramBuilder io.opentelemetry.api.metrics.LongHistogramBuilder.setExplicitBucketBoundariesAdvice(java.util.List)'
	at com.google.cloud.bigquery.storage.v1.ConnectionWorker.registerOpenTelemetryMetrics(ConnectionWorker.java:457) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.ConnectionWorker.<init>(ConnectionWorker.java:511) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.StreamWriter.<init>(StreamWriter.java:258) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.StreamWriter.<init>(StreamWriter.java:59) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.StreamWriter$Builder.build(StreamWriter.java:896) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.SchemaAwareStreamWriter.<init>(SchemaAwareStreamWriter.java:115) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.SchemaAwareStreamWriter.<init>(SchemaAwareStreamWriter.java:57) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.SchemaAwareStreamWriter$Builder.build(SchemaAwareStreamWriter.java:721) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.JsonStreamWriter.<init>(JsonStreamWriter.java:50) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.JsonStreamWriter.<init>(JsonStreamWriter.java:38) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.bigquery.storage.v1.JsonStreamWriter$Builder.build(JsonStreamWriter.java:425) ~[google-cloud-bigquerystorage-3.8.0.jar:3.8.0]
	at com.google.cloud.spring.bigquery.core.BigQueryJsonDataWriter.<init>(BigQueryJsonDataWriter.java:81) ~[spring-cloud-gcp-bigquery-5.5.2-SNAPSHOT.jar:5.5.2-SNAPSHOT]
	at com.google.cloud.spring.bigquery.core.BigQueryTemplate.getBigQueryJsonDataWriter(BigQueryTemplate.java:288) ~[spring-cloud-gcp-bigquery-5.5.2-SNAPSHOT.jar:5.5.2-SNAPSHOT]
	at com.google.cloud.spring.bigquery.core.BigQueryTemplate.getWriteApiResponse(BigQueryTemplate.java:297) ~[spring-cloud-gcp-bigquery-5.5.2-SNAPSHOT.jar:5.5.2-SNAPSHOT]
	at com.google.cloud.spring.bigquery.core.BigQueryTemplate.lambda$writeJsonStream$0(BigQueryTemplate.java:256) ~[spring-cloud-gcp-bigquery-5.5.2-SNAPSHOT.jar:5.5.2-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-6.1.6.jar:6.1.6]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:840) ~[na:na]
```

**Comment by ldetmer** [comment]
This seems to be a similar issue to https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/3011.  This test is running against spring boot 3.2.5 which uses older version of OpenTelemetry that doesn't have the setExplicitBucketBoundariesAdvice.

**Comment by ldetmer** [comment]
fixed in commit [a891693](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/commit/a891693d59c481af23dc2ca44e6b16f200b197fb)
